scripts/ios.rb

```diff
@@ -33,40 +42,32 @@ def link_colocated_native_files(options = {})
   if colocated_files.length > 0
     project = Xcodeproj::Project.open(project_path)
     file_group = project[app_name]
-
-    # check if the "Colocated" group exists
     existing_group = file_group['Colocated']
 
+    remove_nonexistent_files(existing_group, colocated_files)
```

Comment:
This needs to be below

```
# Create the group if it doesn't exist
```

otherwise the group might not exist by the time this function runs.